pkg/command/org_validation_helper_test.go

```diff
@@ -0,0 +1,68 @@
+// Copyright 2024 VMware, Inc. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+package command
+
+import (
+	"errors"
+	"testing"
+
+	"github.com/vmware-tanzu/tanzu-cli/pkg/auth/csp"
+)
+
+func TestValidateTokenForTanzuScopes(t *testing.T) {
+	tests := []struct {
+		name         string
+		claims       *csp.Claims
+		scopesGetter tapScopesGetter
+		expectedBool bool
+		expectedErr  error
+	}{
+		{
+			name:         "NoTAPScopes_Success",
```

Comment:
nit: proper english words in the name field?

Reply:
updated.